experimenter/experimenter/nimbus_ui_new/templatetags/nimbus_extras.py

```diff
@@ -65,3 +67,21 @@ def format_not_set(value):
     if value in [None, "", "NOT SET"]:
         return mark_safe('<span class="text-danger">Not set</span>')
     return value
+
+
+@register.filter(name="format_json")
+def format_json(value):
+    """Formats the JSON value for display by pretty-printing the JSON,
+    and if the JSON is empty or None, it calls format_not_set."""
+    if value in [None, "", "NOT SET"]:
```

Comment:
Also it just occurred to me that you can avoid this and apply multiple filters `  <code>{{ feature_value.value|format_not_set|format_json }}</code>`